InventoryBuilder.py

```diff
@@ -172,13 +174,29 @@ def get_vrops(self):
         elif self.atlas_path:
             try:
                 response = requests.get(url=self.atlas_path)
-                netbox_json = response.json()
-                self.vrops_list = [target['labels']['server_name'] for target in netbox_json if
-                                   target['labels']['job'] == "vrops"]
+                self.response_codes['atlas'][self.atlas_path] = response.status_code
             except (urllib3.exceptions.MaxRetryError, requests.exceptions.ConnectionError,
-                    urllib3.exceptions.NewConnectionError):
-                logger.error(f'Failed to establish a connection to: {self.atlas_path}, retrying in {self.sleep}s')
+                    urllib3.exceptions.NewConnectionError) as err:
+                logger.error(f'Failed to establish a connection to: {self.atlas_path}, retrying in {self.sleep}s', err)
+                if not self.vrops_list:
+                    logger.critical(f'No targets found to start inventory collection, exit 0')
+                    sys.exit(0)
```

Comment:
```suggestion
                    sys.exit(1)
```

Since this is not wanted behaviour I'd rather see it failing instead of gracefully quitting. This way k8s would also recognise, that something is wrong.